From 03c1a77760722ba8b612f7a63ed8b1e6380caffb Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Mon, 10 Oct 2022 04:51:10 +0000
Subject: [PATCH] Revert "recovery: Allow custom bootloader msg offset in block
 misc"

This reverts commit fb8c62bbcda3cdcab7c54bad435defa7dfee2c74.
---
 bootloader_message/Android.bp                          |  1 -
 .../include/bootloader_message/bootloader_message.h    | 10 +++++-----
 2 files changed, 5 insertions(+), 6 deletions(-)

diff --git a/bootloader_message/Android.bp b/bootloader_message/Android.bp
index 23b8e70d..778fdb93 100644
--- a/bootloader_message/Android.bp
+++ b/bootloader_message/Android.bp
@@ -25,7 +25,6 @@ package {
 
 cc_defaults {
     name: "libbootloader_message_defaults",
-    defaults: ["bootloader_message_offset_defaults"],
     srcs: ["bootloader_message.cpp"],
     cflags: [
         "-Wall",
diff --git a/bootloader_message/include/bootloader_message/bootloader_message.h b/bootloader_message/include/bootloader_message/bootloader_message.h
index 45041e53..d58158dd 100644
--- a/bootloader_message/include/bootloader_message/bootloader_message.h
+++ b/bootloader_message/include/bootloader_message/bootloader_message.h
@@ -29,11 +29,11 @@
 // 32K - 64K    System space, used for miscellanious AOSP features. See below.
 // Note that these offsets are admitted by bootloader,recovery and uncrypt, so they
 // are not configurable without changing all of them.
-constexpr size_t BOOTLOADER_MESSAGE_OFFSET_IN_MISC = BOARD_RECOVERY_BLDRMSG_OFFSET;
-constexpr size_t VENDOR_SPACE_OFFSET_IN_MISC = 2 * 1024 + BOARD_RECOVERY_BLDRMSG_OFFSET;
-constexpr size_t WIPE_PACKAGE_OFFSET_IN_MISC = 16 * 1024 + BOARD_RECOVERY_BLDRMSG_OFFSET;
-constexpr size_t SYSTEM_SPACE_OFFSET_IN_MISC = 32 * 1024 + BOARD_RECOVERY_BLDRMSG_OFFSET;
-constexpr size_t SYSTEM_SPACE_SIZE_IN_MISC = 32 * 1024 + BOARD_RECOVERY_BLDRMSG_OFFSET;
+constexpr size_t BOOTLOADER_MESSAGE_OFFSET_IN_MISC = 0;
+constexpr size_t VENDOR_SPACE_OFFSET_IN_MISC = 2 * 1024;
+constexpr size_t WIPE_PACKAGE_OFFSET_IN_MISC = 16 * 1024;
+constexpr size_t SYSTEM_SPACE_OFFSET_IN_MISC = 32 * 1024;
+constexpr size_t SYSTEM_SPACE_SIZE_IN_MISC = 32 * 1024;
 
 /* Bootloader Message (2-KiB)
  *
-- 
2.34.1

